rules.js

```diff
@@ -263,6 +263,6 @@ module.exports = {
     'import/no-amd': ['error'],
     'import/no-commonjs': ['error'],
     'import/no-extraneous-dependencies': ['error'],
-    'prettier/prettier': ['error', { trailingComma: 'es5', singleQuote: true }],
+    'prettier/prettier': ['error', { trailingComma: 'es5', singleQuote: true, printWidth: 120 }],
```

Comment:
ref: https://github.com/prettier/prettier/commit/801519ec79ecaa79e012226ff6b888ea75ba7449

good to know indeed. I am inclined to try 80 and see how it goes.